Your_first_ASDF_converter.ipynb

```diff
@@ -0,0 +1,554 @@
+    "module containing the class, even if it is never used in the program\n",
+    "using ASDF.\n",
+    "\n",
+    "The wildcard for the tag entry indicates that this converter will work\n",
+    "with all versions of the tag. It isn't strictly needed here, but generally \n",
+    "a good practice if one wants the converter code to handle multiple versions.\n",
+    "(How to handle versioning is a topic in its own right and not covered here)\n",
+    "\n",
+    "We need to create a module for the entry point handling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f9c03910",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/mfconverter/extensions.py\n",
+    "from asdf.extension import Extension\n",
+    "from .converter import PhotoIDConverter\n",
+    "\n",
+    "class MFExtension(Extension):\n",
+    "    extension_uri = \"asdf://stsci.edu:example-project/photo_id-1.0.0\"\n",
+    "    tags = [\"asdf://stsci.edu/example-project/photo_id-1.0.0\"]\n",
+    "    converters = [PhotoIDConverter()]\n",
+    "\n",
+    "# The following will be called by ASDF when looking for ASDF entry points    \n",
+    "def get_extensions():\n",
+    "    return [MFExtension()]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "412c3e4f",
+   "metadata": {},
+   "source": [
+    "And finally, the entry point reference in setup.cfg, provided here as a whole file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "feb206d7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile setup.cfg\n",
+    "[metadata]\n",
+    "name = mfconverter\n",
+    "description = TODO\n",
+    "long_description = TODO\n",
+    "author = TODO\n",
+    "version='0.1.0'\n",
+    "license = BSD-3-Clause\n",
+    "\n",
+    "[options]\n",
+    "zip_safe = True\n",
+    "python_requires = >=3.6\n",
+    "setup_requires =\n",
+    "    setuptools_scm\n",
+    "install_requires =\n",
+    "    jsonschema>=3.0.2\n",
+    "    asdf>=2.8\n",
+    "    psutil>=5.7.2\n",
+    "    numpy>=1.16\n",
+    "package_dir =\n",
+    "    =src\n",
+    "packages = find:\n",
+    "\n",
+    "[options.entry_points]\n",
+    "asdf.extensions =\n",
+    "    mfc_extension = mfconverter.extensions:get_extensions\n",
+    "\n",
+    "[options.packages.find]\n",
+    "where = src\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d0e3882",
+   "metadata": {},
+   "source": [
+    "If you wish to learn more about entry points, see:\n",
+    "https://packaging.python.org/guides/creating-and-discovering-plugins/\n",
+    "\n",
+    "We need the setup.py file too"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3ffb22d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile setup.py\n",
+    "#!/usr/bin/env python3\n",
+    "from setuptools import setup\n",
+    "\n",
+    "setup(use_scm_version=False)"
```

Comment:
`use_scm_version=False` is the default so we can just omit that argument

Reply:
Done.